src/main/java/io/vertx/core/streams/WriteStream.java

```diff
@@ -68,6 +70,11 @@ default void end(T t) {
     end();
   }
 
+  default void end(Handler<AsyncResult<Void>> handler) {
```

Comment:
Missing javadoc.

Reply:
I'll address this one when we will reach consensus on whether this method should be added at all.